### DISCUSSION TITLE
Displaying status of untracked files

### DISCUSSION THREAD

**Discussion by yuujay** [airblade/vim-gitgutter#753] [Q&A]
I have airline installed along with gitgutter and fugitive and they are working as expected. However, I have a question with the untracked files. Is there a configuration that I could make use of for setting the status to indicate untracked files? Since, all I am getting now is a status bar that looks like this

![image](https://user-images.githubusercontent.com/3604600/102823994-f1c9cb80-43a9-11eb-8605-613377cde220.png)

I would like to know if there is a way to indicate that the current file is not tracked yet.

**Comment by airblade** [comment]
You can check the variable `b:gitgutter.path`: it will be `-2` if the file is untracked.

I'm sure there's a way to update airline accordingly :)
